circle.yml

```diff
@@ -29,7 +29,7 @@ test:
   pre:
     - python mangaki/manage.py collectstatic --no-input
   override:
-    - coverage run ./mangaki/manage.py test
+    - cd mangaki && coverage run manage.py test
```

Comment:
We need to move `.coveragerc` and similar files to the `mangaki` directory now, else, it won't be taken in account.